index.html

```diff
@@ -587,75 +589,89 @@ <h4>
-      (a <a>Text record</a>), one or more icon records, a type record, a size
-      record and an action record.
-    </p>
-    <p>
-      Icon records are <a>MIME type record</a>s. If multiple icon records are
-      included, readers SHOULD select only one of them to display.
-    </p>
-    <p>
-      The type record has [=local type name=] "`t`" specific to smart
-      poster and the <a>PAYLOAD field</a> contains a UTF-8 encoded MIME type for
-      the content referred to by the URI record.
-    </p>
-    <p>
-      The size record has [=local type name=] "`s`" specific to smart
-      poster and the <a>PAYLOAD field</a> contains a 4-byte 32 bit unsigned
-      integer that denotes the size of the object referred to by the URL in the
-      URI record.
-    </p>
-    <p>
-      The action record has [=local type name=] "`act`" specific to
-      smart poster and the <a>PAYLOAD field</a> contains a single byte, whose
-      value has the following meaning:
-    </p>
-    <table class="simple">
-      <tr>
-        <th><strong>Value</strong></th>
-        <th><strong>Description</strong></th>
-      </tr>
-      <tr>
-        <td>0</td>
-        <td>Do the action</td>
-      </tr>
-      <tr>
-        <td>1</td>
-        <td>Save for later</td>
-      </tr>
-      <tr>
-        <td>2</td>
-        <td>Open for editing</td>
-      </tr>
-      <tr>
-        <td>3..0xFF</td>
-        <td>Reserved for future use</td>
-      </tr>
-    </table>
-    <p>
-      The action record is optional and there is no default action on the smart
-      poster content if the action record is missing.
-    </p>
-    <p class="note">
-      At the time of NDEF standardization the value `0` ("do the action") was
-      meant for use cases like send an SMS, make a call or launch browser.
-      Similarly, the value `1`, ("save the content for later processing") was
-      meant for use cases like store the SMS in inbox, save the URL in
-      bookmarks, or save the phone number to contacts. Also, the value `2`
-      ("open for editing") was meant to open the smart poster content with a
-      default application for editing.
-    </p>
-    <p>
-      Implementations don't need to implement any standardized behavior for the
-      actions defined here. In this API it's up to the applications what actions
-      they define (that may include the use cases above). However, Web NFC
-      just provides the values.
-    </p>
+    <div>
+      <dfn>Smart poster</dfn> is defined in [[NDEF-SMARTPOSTER]] to describe a
+      given Web content as an NDEF record that contains an <a>NDEF message</a>
+      as payload, including the following records:
+      <ul>
+        <li>
+          A single mandatory <a>URI record</a> that refers to the
+          <a>smart poster</a> content.
+        </li>
+        <li>
+          Zero or more <a>Text records</a> that act as a <dfn>title record</dfn>
+          related to the content. When there are more than one title records
```

Comment:
s/records/record/ ?

Reply:
Not sure about "web content". English grammar seems to prefer "Web content". At least it's disputed. But indeed if we can say "we" use "web content", that's what "we" use I guess :).

Reply:
US English dictionaries changed... it used to be e-mail, now email is preferred and it used to be Web always, but now web is totally fine.